php/example_code/sns/README.md

```diff
@@ -0,0 +1,35 @@
+# Amazon Simple Notification Service (SNS) AWS SDK for PHP - Version 3 code examples
+
+## Purpose
+The code examples in this directory demonstrate how to work with the  Amazon Simple Notification Service 
+(Amazon SNS) using the AWS SDK for PHP version 3 (v3).
```

Comment:
PHP Version 3
(capitalize "Version" to match dev guide, remove (v3) (seems unnecessary))